### PR TITLE
changed description and pared down expectations

### PR DESCRIPTION
**Pull request by itsdrewmiller**
@ojintoad

### CODE REVIEW

File: advertisements/devops engineer.md [hunk 1]
```diff
@@ -5,32 +5,26 @@ DevOps Engineer
 Responsibilities
 ----------------
 
-We’re seeking a self-driven DevOps Administrator who will plan, implement, and maintain deployment solutions.  With a focus on working with high level engineers and IT staff, candidates will provide a seamless integration point for Development and Information Technology.  Candidates should have the ability to recommend and rapidly deploy solution-based systems, automate manual processes, and work well on cross-functional teams.
+The goal of our DevOps team is to automate all the work that goes into turning an idea into a production feature, to the extent allowed by Descartes and Heisenberg.  It turns out that we do a lot of things that would be faster, cheaper, and safer if automated, and we'd like to hire someone to help us make even more progress on them.  The job itself involves a fair bit of research to understand the engineering problems and find good technical solutions, as well as plenty of scripting in Powershell.
```
Comment (ojintoad) [inline advertisements/devops engineer.md]: Per dm - not super thrilled with Descarates and Heisenberg as it comes off more snooty than fun but not against the joke.

File: advertisements/devops engineer.md [hunk 2]
```diff
@@ -5,32 +5,26 @@ DevOps Engineer
...
 What we're looking for
 ----------------------
 
 You don't have to fit all of these bullets, but if you don't fit any of them this is probably not the right job for you.
 
-* Experience with Linux distributions and maintenance
-* Experience with scripting languages (Bash, PowerShell, etc.)
+* Experience with scripting languages (Especially Powershell)
 * Experience with and understanding of networking, DNS, load-balancing, and clustering technologies
 * Experience with continuous integration/continuous deployment (TeamCity)
```
Comment (ojintoad) [inline advertisements/devops engineer.md]: Whey not Especially TeamCity to stay in line

File: advertisements/devops engineer.md (hunk 1 above)
Comment (itsdrewmiller) [reply advertisements/devops engineer.md]: I got rid of it.  RELUCTANTLY

**Comment by itsdrewmiller** [comment]
Merging this to get it up on Hacker News - still can iterate a bit before we fully syndicate in jazz

**Pull request merged**